index.html

```diff
@@ -30,124 +30,122 @@ <h1>The TC39 Process</h1>
-      <th>Post-Acceptance Changes Expected
-      <th>Implementation Types Expected*
+      <th style="cursor: help" title="These are the actions that will be undertaken by the committee and the proposal authors while the proposal is in this stage.">Purpose
     </tr>
   </thead>
+
   <tr>
     <td>0
-    <td>Allow input into the specification
-    <td>None
-    <td>N/A
-    <td>N/A
-    <td>N/A
-    <td>N/A
-  </tr>
-  <tr>
-    <td>1
+    <td>This is a new proposal. It is not currently being considered by the committee.
+    <td>None. New proposals are assigned this stage by their authors outside of the usual advancement process.
     <td>
+      Ideation and exploration. Define a problem space in which the committee and the champions can focus their efforts.
       <ul>
-        <li>Make the case for the addition
-        <li>Describe the shape of a solution
+        <li>Make the case for an improvement
+        <li>Describe the shape of some possible solutions
         <li>Identify potential challenges
+        <li>Research how the problem is dealt with using available facilities today
+        <li>Research how the problem has been solved by other languages or in the library ecosystem
       </ul>
-    </td>
+  </tr>
+
+  <tr>
+    <td>1
+    <td>This proposal is under consideration. The committee expects to devote time to examining the identified problem space, the full breadth of possible solutions, and cross-cutting concerns. 
     <td>
       <ul>
-        <li>Identified “champion” who will advance the addition
+        <li>Identified a champion or champion group who will advance the addition
         <li>Prose outlining the problem or need and the general shape of a solution
-        <li>Illustrative examples of usage
-        <li>High-level API
         <li>Discussion of key algorithms, abstractions and semantics
-        <li>Identification of potential “cross-cutting” concerns and implementation challenges/complexity
+        <li>Identification of potential cross-cutting concerns and implementation challenges/complexity
         <li>A publicly available repository for the proposal that captures the above requirements
       </ul>
     </td>
-    <td>The committee expects to devote time to examining the problem space, solutions and cross-cutting concerns
+    <td>
+      Designing a solution.
+      <ul>
+        <li>Make the case for a particular solution or solution space
+        <li>Resolve any cross-cutting concerns
+      </ul>
     </td>
-    <td>None
-    <td>Major
-    <td>Polyfills / demos
   </tr>
+
   <tr>
     <td>2
-    <td>Precisely describe the syntax and semantics using formal spec language
+    <td>The committee has chosen a preferred solution or solution space, but the design is a draft and may still change significantly. The committee expects the feature to be developed and eventually included in the standard, but due to reasons that may not be apparent at this stage, the feature may never advance.
```

Comment:
> but due to reasons that may not be apparent at this stage, the feature may never advance.

wording sounds a little strange to me, but I don't immediately have a suggestion to improve

Reply:
Well, this isn't really actionable for me. The wording doesn't sound strange to me, so until you have a suggested alternative, marking as resolved.

Reply:
> reasons that may not be apparent at this stage

if the proposal will not advance to the next stage, that suggests that the reasons _will_ be apparent at this stage

Reply:
so thinking about this a bit more, is what we're really saying here: due to reasons that may not be apparent at this stage (stage 2), the feature may never advance (to stage 3), and it first needs to go to stage 2.7 for those reasons to become apparent

Reply:
That is one possibility. The original wording that I was trying to preserve was "The committee expects the feature to be developed and eventually included in the standard". Since this column is intended to target the community, I wanted to moderate the wording here as to not convey a commitment (through the use of "expects") to proposals at stage 2. Anything short of making it to stage 4 could be seen as a failure of this commitment. So I modified it to what you see here.

Reply:
```suggestion
    <td>The committee has chosen a preferred solution or solution space, but the design is a draft and may still change significantly. The committee expects the feature to be developed and eventually included in the standard, but due to reasons that may not be apparent at this stage, the feature may never be included in the standard.
```

?

Reply:
Sure, done.